---a/net/minecraft/enchantment/EnchantmentArrowInfinite.java
+++b/net/minecraft/enchantment/EnchantmentArrowInfinite.java
@@ -2,29 +2,25 @@
 
 import net.minecraft.util.ResourceLocation;
 
-public class EnchantmentArrowInfinite extends Enchantment
-{
+public class EnchantmentArrowInfinite extends Enchantment {
+
     private static final String __OBFID = "CL_00000100";
 
-    public EnchantmentArrowInfinite(int enchID, ResourceLocation enchName, int enchWeight)
-    {
+    public EnchantmentArrowInfinite(int enchID, ResourceLocation enchName, int enchWeight) {
         super(enchID, enchName, enchWeight, EnumEnchantmentType.BOW);
         this.setName("arrowInfinite");
     }
 
-    public int getMinEnchantability(int enchantmentLevel)
-    {
+    public int getMinEnchantability(int enchantmentLevel) {
         return 20;
     }
 
-    public int getMaxEnchantability(int enchantmentLevel)
-    {
+    public int getMaxEnchantability(int enchantmentLevel) {
         return 50;
     }
 
-    public int getMaxLevel()
-    {
+    public int getMaxLevel() {
         return 1;
     }
-}
+}
